services/cd-service/pkg/repository/transformer_db_test.go

```diff
@@ -648,6 +648,101 @@ func TestTeamLockTransformersWithDB(t *testing.T) {
 	}
 }
 
+func TestCreateApplicationVersionDBRevisions(t *testing.T) {
+	const appName = "app1"
+	tcs := []struct {
+		Name               string
+		Transformers       []Transformer
+		expectedDbContent  *db.DBAppWithMetaData
+		expectedDbReleases []types.ReleaseNumbers
+	}{
+		{
+			Name: "create two versions, same version, different revision",
+			Transformers: []Transformer{
+				&CreateEnvironment{
+					Environment: envAcceptance,
+					Config:      config.EnvironmentConfig{Upstream: &config.EnvironmentConfigUpstream{Environment: envAcceptance, Latest: false}},
+				},
+				&CreateApplicationVersion{
+					Application: appName,
+					Version:     10,
+					Revision:    1,
+					Manifests: map[types.EnvName]string{
+						envAcceptance: "{}",
+					},
+					Team: "t1",
+				},
+				&CreateApplicationVersion{
+					Application: appName,
+					Version:     10,
+					Revision:    2,
+					Manifests: map[types.EnvName]string{
+						envAcceptance: "{test}",
+					},
+					Team: "t1",
+				},
```

Comment:
I'd like to see another test case with the same input, except the revisions are swapped.
The expected results should be exactly the same.